src/main/java/org/folio/rest/impl/AbstractHelper.java

```diff
@@ -95,6 +106,54 @@ public void closeHttpClient() {
     return processingErrors.getErrors();
   }
 
+  protected boolean isCheckin(PoLine poLine) {
+    return defaultIfNull(poLine.getCheckinItems(), false);
```

Comment:
The [schema defines default value](https://github.com/folio-org/acq-models/blob/master/mod-orders-storage/schemas/po_line.json#L18) so I think this method is not needed.

Reply:
Done, few tests had to be refactored.